docusaurus/docs/components/analytics/analytics.md

```diff
@@ -46,69 +62,86 @@ entries.event: click
 entries.url: <url of the page where the user performed the action>
 ```
 
-#### Live example: [Storybook](https://availity.github.io/availity-react/storybook/?path=/story/components-analytics--default)
+NOTE: In the code sample above, the Reactstrap `<Button>` component is able forwarded these data attributes to the native `<button>` element that the component renders. Not all React components can accept these data analytics attributes. Use the Elements panel of DevTools to confirm that these attributes are added to the appropriate DOM element.
+
+Be sure to read more on auto tracking and the required data attributes [here on the docs for the AvAnalytics class](https://availity.github.io/sdk-js/resources/analytics/#auto-tracking-with-data-analytics-attributes).
+
+### Manual Tracking
+
+If the [limitations of auto tracking](https://availity.github.io/sdk-js/resources/analytics/#limitations) prevent you from logging the events you are interested in, you will need to use manual tracking by calling analytics.trackEvent() directly. For this, you will need access to the instance of AvAnalytics that has been initialize by the Analytics context provider. This instance is return by the [useAnalytics hook](/availity-react/components/analytics/hook).
```

Comment:
`The instance is returned` instead of `return`

Reply:
Fixed. Thanks!